doc/batch_settings.rst

```diff
@@ -0,0 +1,110 @@
+**************
+Batch Settings
+**************
+
+========
+Overview
+========
+SmartSim offers functionality to launch workloads (``Model`` or ``Ensemble``)
+as batch jobs. Each SmartSim `launcher` interfaces with a ``BatchSettings`` object
+specific to a systems Workload Manager (WLM). The following ``BatchSettings`` child
+classes are provided per `launcher`:
+
+1. The Slurm `launcher` supports:
+   - :ref:`SbatchSettings<sbatch_api>`
+2. The PBSpro `launcher` supports:
+   - :ref:`QsubBatchSettings<qsub_api>`
+3. The Cobalt `launcher` supports:
+   - :ref:`CobaltBatchSettings<cqsub_api>`
+4. The LSF `launcher` supports:
+   - :ref:`BsubBatchSettings<bsub_api>`
+
+Each above child class allow you to specify the run parameters for a WLM batch job.
+The local `launcher` does **not** support batch jobs.
+
+Once a ``BatchSettings`` object is initialized via the ``Experiment.create_batch_settings()``
+function, you have access to helper functions associated with the object that allow
+configuring the job batch settings. The following chart illustrates helper functions
+provided by the ``SrunSettings`` object. The ``SbatchSettings`` object is used for
+launching batches on Slurm WLM systems.
+
+.. list-table:: SbatchSettings Helper Functions
+   :widths: 25 55 25
+   :header-rows: 1
+
+   * - ``SbatchSettings`` func
```

Comment:
let's not abbreviate "func" to function... 

nitpick: _technically_ i think we refer to functions as functions and class members as methods, so you could be pedantic and replace function w/method